packages/effector-react/README.md

```diff
@@ -19,29 +19,27 @@ yarn add effector effector-react
 ```js
 import {createStore, combine, createEvent} from 'effector'
 
-import {useStore} from 'effector-react'
+import {useUnit} from 'effector-react'
 
 const inputText = createEvent()
 
 const $text = createStore('')
   .on(inputText, (_, text) => text)
   
-const $size = createStore(0)
-  .on(inputText, (_, text) => text.length)
-
-const $form = combine({
-  text: $text,
-  size: $size,
-})
+const $size = $text.map(text => text.length)
 
 const Form = () => {
-  const {text, size} = useStore($form)
+  const {text, size} = useUnit({
+    text: $text,
+    size: $size,
+  })
+  const [handleTextChange] = useUnit([inputText])
```

Comment:
Why not just `useUnit(inputText)` ?

Reply:
I wanted to show that different kinds of overloads are supported 🤔

Reply:
I think we have explanation in the docs, but readme is just and example, isn't it?